.github/instructions/dsc-community-style-guidelines-command-integration-tests.instructions.md

```
@@ -0,0 +1,61 @@
+environment and it will make sure the module being tested is available
+
+```powershell
+[System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', '', Justification = 'Suppressing this rule because Script Analyzer does not understand Pester syntax.')]
+param ()
+
+BeforeDiscovery {
+    try
+    {
+        if (-not (Get-Module -Name 'DscResource.Test'))
+        {
+            # Assumes dependencies has been resolved, so if this module is not available, run 'noop' task.
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 80; Actual: 106

(MD013, line-length)